src/facetgroups/MultiselectDropdownFacetGroup.js

```diff
@@ -48,15 +46,16 @@ const ebiVfSelectStyles = {
   })
 }
 
-const MultiselectDropdownFacetGroup = ({hideName, facetGroupName, facets, onChange}) =>
-  <div className={styles[`input-clear`] + ` padding-bottom-xlarge`}>
-    {!hideName && <h4>{facetGroupName}</h4>}
+const MultiselectDropdownFacetGroup = ({facetGroupName, facets, onChange}) =>
+  <div className={`padding-bottom-xlarge`}>
+    <h4>{facetGroupName}</h4>
     <Select components={{ DropdownIndicator, IndicatorSeparator: null }}
             styles={ebiVfSelectStyles}
             closeMenuOnSelect={false}
             isMulti={true}
             onChange={(args) => onChange(facetGroupName, args)}
-            options={facets} />
+            // isOptionDisabled={(option) => option.disabled}
```

Comment:
I’m leaving it there because I’d like to show it in a group meeting. Disabled options are left there but greyed out. It might be better than to just hide unavailable facets.